plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduConnectorTest.java

```diff
@@ -510,17 +510,12 @@ protected String errorMessageForInsertNegativeDate(String date)
     }
 
     @Override
-    public void testInsertRowConcurrently()
+    protected TestTable createTableWithOneIntegerColumn(String namePrefix)
```

Comment:
Move to after `verifyConcurrentAddColumnFailurePermissible()`.

Reply:
@ebyhr Fixed :) 
(I assume you mean the `createTableWithOneIntegerColumn` in `BaseConnectorTest` because `verifyConcurrentAddColumnFailurePermissible` is in `BaseConnectorTest`)